ui/raidboss/data/04-sb/dungeon/bardams_mettle.txt

```diff
@@ -0,0 +1,169 @@
+1038.9 "Charge" sync /:Throwing Spear:257F:/
+1044.6 "--sync--" sync /:Bardam:24EA:/ window 15,15
+1052.3 "Travail" sync /:Bardam:1EFF:/
+1060.5 "Sacrifice" sync /:Bardam:1F01:/
+1060.7 "Travail" sync /:Bardam:1EFF:/
+1065.5 "Bardam's Ring" sync /:Bardam:2581:/
+1066.2 "Travail" sync /:Bardam:1EFF:/
+1076.4 "Comet x8" duration 12 # sync /:Bardam:257D:/
+1093.4 "Heavy Strike x3" sync /:Hunter Of Bardam:2577:/ duration 4
+1099.0 "Travail" sync /:Bardam:1EFF:/
+1105.3 "Comet Impact" sync /:Star Shard:2580:/
+1109.3 "--sync--" sync /:Bardam:24EA:/ window 15,15
+1116.9 "Travail" sync /:Bardam:1EFF:/
+1125.3 "Reconstruct" sync /:Bardam:1EFD:/
+1126.7 "--sync--" sync /:Star Shard:258B:/
+1128.3 "Travail" sync /:Bardam:1EFF:/
+1133.1 "Charge" sync /:Throwing Spear:257F:/
+1133.9 "Magnetism" sync /:Hunter Of Bardam:1F08:/
+1139.5 "Tremblor" sync /:Hunter Of Bardam:2585:/
+1140.0 "Heavy Strike x3" sync /:Warrior Of Bardam:2577:/ duration 4
+1153.0 "Meteor Impact" sync /:Looming Shadow:2582:/
+1153.4 "--sync--" sync /:Star Shard:258B:/
+
+#~~~~~#
+# YOL #
+#~~~~~#
+
+# -ii 1F0B 1F0E
+
+2000.0 "Start" sync / 00:0839:.*Voiceless Muse will be sealed off/ window 2000,5
+2008.1 "Feathercut" sync /:Yol:1F09:/
+2015.2 "Wind Unbound" sync /:Yol:1F0A:/
+2023.9 "Pinion" sync /:Yol Feather:1F11:/
+2025.8 "Flutterfall" sync /:Yol:1F0C:/
+2027.2 "Feathercut" sync /:Yol:1F09:/
+2033.9 "Pinion" sync /:Yol Feather:1F11:/
+2034.7 "Flutterfall" sync /:Yol:1F0C:/
+2042.2 "Wind Unbound" sync /:Yol:1F0A:/
+2046.4 "Feathercut" sync /:Yol:1F09:/
+2050.9 "Pinion" sync /:Yol Feather:1F11:/
+2054.0 "Eye Of The Fierce" sync /:Yol:1F0D:/
+2059.2 "Feathercut" sync /:Yol:1F09:/
+2060.9 "Pinion" sync /:Yol Feather:1F11:/
+2069.3 "Wind Unbound" sync /:Yol:1F0A:/
+2071.4 "--untargetable--"
+2071.4 "--adds spawn--" sync /03:........:Added new combatant Corpsecleaner Eagle/ window 71.4,5
+2078.0 "Pinion" sync /:Yol Feather:1F11:/
+2088.0 "Pinion" sync /:Yol Feather:1F11:/
+2098.0 "Pinion" sync /:Yol Feather:1F11:/
+2108.0 "Pinion" sync /:Yol Feather:1F11:/
+2118.0 "Pinion" sync /:Yol Feather:1F11:/ jump 2078.0
+2128.0 "Pinion"
+2138.0 "Pinion"
+2148.0 "Pinion"
+
+# Unfortunately, because there are two Corpsecleaner Eagles,
+# we can't just sync to a 04 log line.
+2189.7 "--targetable--" sync /22:........:Yol:........:Yol:01/ window 120,5
+2196.8 "Feathercut" sync /:Yol:1F09:/
+2204.0 "Wind Unbound" sync /:Yol:1F0A:/
+2212.7 "Pinion" sync /:Yol Feather:1F11:/
+2214.5 "Flutterfall" sync /:Yol:1F0C:/
+2215.8 "Feathercut" sync /:Yol:1F09:/
+2222.6 "Pinion" sync /:Yol Feather:1F11:/
+2223.3 "Flutterfall" sync /:Yol:1F0C:/
+2230.7 "Wind Unbound" sync /:Yol:1F0A:/
+2234.9 "Feathercut" sync /:Yol:1F09:/
+2239.4 "Pinion" sync /:Yol Feather:1F11:/
+2242.5 "Eye Of The Fierce" sync /:Yol:1F0D:/
+2247.7 "Feathercut" sync /:Yol:1F09:/
+2249.4 "Pinion" sync /:Yol Feather:1F11:/
+
+2253.8 "--untargetable--" sync /22:........:Yol:........:Yol:00/ window 60,5
```

Comment:
Is this also a fixed time? for some reason I thought it was an hp push

Reply:
This is fixed as well. I tested both with the boss at 90%+ HP and with bringing the boss to 10% HP before this point in the rotation, and it happened at the same point both times. It's *possible* there's a push, but it seems unlikely.